cl/corpus_importer/tests.py

```diff
@@ -2777,6 +2777,12 @@ def test_merge_overlap_judges(self):
                 "Leavitt",
                 "",  # extracted data is the same, no need to update
             ),
+            # CL item #1301211
+            (
+                "Mikell",
+                "MlKELL",  # there is a typo in the name, but it is very similar, we shouldn't be throwing an exception
+                "Mikell",
+            ),
```

Comment:
what is our default here?  the default is to go with what we have over the harvard data because it is OCRd?

Reply:
For the moment, for the judges I assumed that the information that is currently in CL is better than the information from Harvard, In the example i added, the name that is incorrectly written comes from Harvard, the correct one is in cl.

https://ia903105.us.archive.org/10/items/law.free.cap.ga-app.278/650.3740654.json

Reply:
great.  I think thats right.